rabbitmq/datadog_checks/rabbitmq/rabbitmq.py

```diff
@@ -531,12 +531,14 @@ def _get_metrics(self, data, object_type, custom_tags):
             # Walk down through the data path, e.g. foo/bar => d['foo']['bar']
             root = data
             keys = attribute.split('/')
+
+            # In RabbitMQ 3.1.x queue_totals is an empty list instead of a dict when initialising
```

Comment:
We don't want to handle the case for 3.1 ?

If yes, I would understand since 3.1 end of life is 2014: https://www.rabbitmq.com/versions.html

Reply:
Ok, just read https://github.com/DataDog/integrations-core/pull/4805#issuecomment-543575023

Maybe a bug on rabbitmq for `queue_totals` initialisation ?